collections.md

```diff
@@ -90,27 +90,6 @@ We can use foreach collection items index as serial no (SL) in pagination.
 
 it will solve the issue of next pages(?page=2&...) index count from continue.
 
```

Comment:
This deletion is not mentioned in your commit or PR. It is similar but different from the tip that follows it and should probably be kept.

```suggestion

### Higher order collection methods

Collections have higher order methods, this are methods that can be chained , like `groupBy()` , `map()` ... Giving you a fluid syntax. This example calculates the
price per group of products on an offer.

```php
$offer = [
        'name'  => 'offer1',
        'lines' => [
            ['group' => 1, 'price' => 10],
            ['group' => 1, 'price' => 20],
            ['group' => 2, 'price' => 30],
            ['group' => 2, 'price' => 40],
            ['group' => 3, 'price' => 50],
            ['group' => 3, 'price' => 60]
        ]
];

$totalPerGroup = collect($offer['lines'])->groupBy('group')->map(fn($group) => $group->sum('price'));
```
```